test/libsolidity/syntaxTests/structs/member_type_eq_name.sol

```diff
@@ -3,4 +3,4 @@ contract C {
   function f(function(S memory) external) public {}
 }
 // ----
-// TypeError 5172: (25-26): Name has to refer to a struct, enum or contract.
+// DeclarationError 7920: (25-26): Identifier not found or not unique.
```

Comment:
Isn't type error the correct error to output here?

Reply:
Could be, but then it would be the only type error in `ReferencesResolver`.